testsuite/features/secondary/srv_cobbler_buildiso.feature

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+@scope_cobbler @scope_iso
+Feature: Cobbler buildiso
+
+  Scenario: Log in as testing user in the cobbler buildiso context
+    Given I am authorized as "testing" with password "testing"
+
+  Scenario: Copy cobbler profiles on the server in the cobbler buildiso context
+    When I copy autoinstall mocked files on server
```

Comment:
I wonder if we should make this step more specific for cobbler, haven't we other mechanism for autoinstall?

Reply:
We already had it in the testsuite, so I copied it from https://github.com/uyuni-project/uyuni/blob/57ae43ccf9750f42190cfcd160f7d6251b39d4a5/testsuite/features/secondary/srv_distro_cobbler.feature#L10-L11

Reply:
OK, then fine to don't change it for this PR.